### PR TITLE
[17.0][MIG]account_fiscal_year:Migration to 17.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Review by yankinmax — APPROVED** [review]

**Review by vincent-hatakeyama — APPROVED** [review]

**Comment by salvorapi** [comment]
LGTM @fclementic2c

File: account_fiscal_year/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Copyright 2020 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Account Fiscal Year",
+    "summary": "Create Account Fiscal Year",
+    "version": "17.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Accounting",
+    "website": "https://github.com/OCA/account-financial-tools"
+    "14.0/account_fiscal_year",
```
Comment (eLBati) [inline account_fiscal_year/__manifest__.py]: `17.0` or `https://github.com/OCA/account-financial-tools` even better

**Review by eLBati — CHANGES_REQUESTED** [review]

**Comment by peluko00** [comment]
Can you push again for recreate build @bizzappdev please

File: account_fiscal_year/__manifest__.py (hunk 1 above)
Comment (bizzappdev) [reply account_fiscal_year/__manifest__.py]: changed to https://github.com/OCA/account-financial-tools only.

**Comment by bizzappdev** [comment]
@peluko00 Done and build also passed

**Review by peluko00 — APPROVED** [review]
LGTM, tested in runboat

**Comment by peluko00** [comment]
> @peluko00 Done and build also passed

Thanks!

**Review by mpascuall — APPROVED** [review]
Working good, checked in runboat

**Review by eLBati — APPROVED** [review]
Thanks

**Review by salvorapi — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

Offtopic: This module was much better when it used to use date ranges :cry:

**Pull request merged**